### ISSUE TITLE
Unable to horizontal center image on top of slide

### ISSUE THREAD

**Issue by gustavsson15** [nicinabox/superslides#228]
Hi, i want an image to be horizontal  centered on top of the slide, look at image1 (the black box). I've tried to use " margin-left: auto; margin-right:auto; " but then the box just disappear.

i've attached screenshot of html and css.

![image1](https://cloud.githubusercontent.com/assets/7395765/2790173/237967ce-cbb8-11e3-89fb-5ca906b506d1.png)
![image2](https://cloud.githubusercontent.com/assets/7395765/2790175/237c0bf0-cbb8-11e3-93ba-c4ae666c80bd.png)
![image3](https://cloud.githubusercontent.com/assets/7395765/2790174/237bcc62-cbb8-11e3-9096-2bc85fec88ee.png)

pleeeeease help me! :)

**Comment by nicinabox** [comment]
Add `.preserve` to the images that aren't backgrounds.

**Comment by gustavsson15** [comment]
Hi, thank you for the answer, but i don't understand what you mean, where and how should i add ".preserve"?

i'm not the king of css/html but i hope this could be solved haha! :)

appreciate the help!

**Comment by nicinabox** [comment]
`<img class="preserve" ... />`

This will prevent superslides from treating it as a background image, so you can style it as you like.

**Comment by gustavsson15** [comment]
hm, still doesn't work.. is something wrong in attached files?

Thank you.

![skarmavbild 2014-04-25 kl 15 38 33](https://cloud.githubusercontent.com/assets/7395765/2801199/2b5d505a-cc7f-11e3-9a47-2c4bc5d7a1e2.png)
![skarmavbild 2014-04-25 kl 15 38 53](https://cloud.githubusercontent.com/assets/7395765/2801200/2b5d5c6c-cc7f-11e3-9995-3d974229db9d.png)
![skarmavbild 2014-04-25 kl 15 39 06](https://cloud.githubusercontent.com/assets/7395765/2801198/2b5d2fa8-cc7f-11e3-9b0f-d7a667e4ccf3.png)

**Comment by nicinabox** [comment]
Yes, `#box` cannot have both `position:absolute` and `margin: auto`. Either use margins, or use absolute positioning.

**Comment by gustavsson15** [comment]
Thank you for the help, didn't got it to work anyway though, if i removed "position: absolut;" the box disapeared. But i solved it with "left: 50%;" and "margin-left: -150px;" (the box is 300px).

confused why it doesn't work in the correct way but thank you for your time!

**Comment by nicinabox** [comment]
Because you probably need some relative positioning in order to control the z-index. And that’s fine.
